specification/purview/data-plane/Azure.Purview.Catalog/preview/2020-12-01-preview/purviewcatalog.json

```diff
@@ -0,0 +1,6748 @@
+          },
+          {
+            "$ref": "#/parameters/offset"
+          },
+          {
+            "$ref": "#/parameters/limit"
+          },
+          {
+            "$ref": "#/parameters/apiVersionNonAtlas"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK. If Lineage exists for the given entity.",
+            "schema": {
+              "$ref": "#/definitions/AtlasLineageInfo"
+            }
+          }
+        }
+      }
+    },
+    "/atlas/v2/relationship": {
+      "post": {
+        "tags": [
+          "RelationshipREST"
+        ],
+        "summary": "Create a new relationship between entities.",
+        "description": "Create a new relationship between entities.",
+        "operationId": "Relationship_Create",
+        "x-ms-examples": {
+          "RelationshipREST_Create": {
+            "$ref": "./examples/RelationshipREST_Create.json"
+          }
+        },
+        "parameters": [
+          {
+            "in": "body",
+            "name": "relationship",
+            "description": "The AtlasRelationship object containing the information for the relationship to be created.",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/AtlasRelationship"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK. The relationship is created.",
+            "schema": {
+              "$ref": "#/definitions/AtlasRelationship"
+            }
+          }
+        }
+      },
+      "put": {
+        "tags": [
+          "RelationshipREST"
+        ],
+        "summary": "Update an existing relationship between entities.",
+        "description": "Update an existing relationship between entities.",
+        "operationId": "Relationship_Update",
+        "x-ms-examples": {
+          "RelationshipREST_Update": {
+            "$ref": "./examples/RelationshipREST_Update.json"
+          }
+        },
+        "parameters": [
+          {
+            "in": "body",
+            "name": "relationship",
+            "description": "The AtlasRelationship object containing the information for the relationship to be created.",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/AtlasRelationship"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK. The relationship is updated.",
+            "schema": {
+              "$ref": "#/definitions/AtlasRelationship"
+            }
+          }
+        }
+      }
+    },
+    "/atlas/v2/relationship/guid/{guid}": {
+      "get": {
+        "tags": [
+          "RelationshipREST"
+        ],
+        "summary": "Get relationship information between entities by its GUID.",
+        "description": "Get relationship information between entities by its GUID.",
+        "operationId": "Relationship_GetById2",
```

Comment:
Why is there a trailing "2"?

Reply:
After checking the Atlas swagger, I will move this "2" here.